include/gmapping/grid/array2d.h

```diff
@@ -7,6 +7,10 @@
 
 #include <iostream>
 
+#ifdef _WIN32
+#define __PRETTY_FUNCTION__ __FILE__":" << __LINE__
```

Comment:
Maybe `__FUNCSIG__ ":" << __LINE__` instead of using typeid. (`__FUNCTION__` to get only the function name or `__FUNCDNAME__` to get the decorated one).